python/hsfs/feature_group_commit.py

```diff
@@ -67,6 +71,8 @@ def to_dict(self):
             "rowsUpdated": self._rows_updated,
             "rowsDeleted": self._rows_deleted,
             "validationId": self._validation_id,
+            "archived": self._archived,
+            "lastActiveCommitTime": self._last_active_commit_time,
```

Comment:
In the backend you said it's currently not being sent to the client?

Reply:
Yes the backend doesn't return it, but the client sends it and the objects are the same.